src/Symfony/Component/EventDispatcher/EventListener.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\EventDispatcher;
+
+/**
+ * Helper class for single-event subscription.
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+abstract class EventListener implements EventSubscriberInterface
```

Comment:
what about `SingleEventListener`?

Reply:
an event listener is "single" already, otherwise it be an event subscriber :)

Reply:
oki thanks for the tip :)